src/lib.rs

```diff
@@ -41,6 +45,48 @@ use structopt::StructOpt;
 ///
 /// - `authors`: Author names, taken from usual environment variables (i.e.
 ///   those which are also used by Cargo and git)
+///
+/// The template author can define their own placeholders in their
+/// `cargo-generate.toml` file. This looks like the following:
+///
+/// [placeholders]
```

Comment:
this embedded code needs to be wrapped inside a `code` block that starts with `` ```toml ``

you can verify that it works via `cargo doc --verbose` then you see the warnings.

Reply:
Yeah, right

The issue is how it looks like in the generated help message, but I'll come up with something

Reply:
I suggest we move it all to a module level comment `//!` then it appears on the docs quite nicely.

Reply:
Right, but then structop isn't able to pick it up as the help message, right? Am I missing something?

Reply:
I'm not super familiar with `structops` but I was under the impression that the help messages comes directly from the comments above the individual fields. But I can double-check it.

Reply:
AFAIK, it gets the help message from the struct documentation itself for the general `-h` message

You can see it if you just do `cargo run` with no extra parameters